jetty-core/jetty-maven/src/main/java/org/eclipse/jetty/maven/AbstractJettyEmbedder.java

```diff
@@ -271,14 +260,13 @@ protected void applyWebAppProperties() throws Exception
         }
     }
 
-    private void configureShutdownMonitor()
+    private void configureServerShutdown(Server server)
     {
         if (stopPort > 0 && stopKey != null)
         {
-            ShutdownMonitor monitor = ShutdownMonitor.getInstance();
-            monitor.setPort(stopPort);
-            monitor.setKey(stopKey);
-            monitor.setExitVm(exitVm);
+            ServerShutdown serverShutdown = new ServerShutdown("127.0.0.1", stopPort, stopKey, exitVm);
```

Comment:
I don't think this is correct - the `stopPort`, `stopKey`, `exitVm` have setters on the embedder instance, they aren't coming from jetty properties.